### DISCUSSION TITLE
有谁知道DNS模式里的normal有什么用吗？

### DISCUSSION THREAD

**Discussion by xiandu999-yu** [MetaCubeX/mihomo#1343] [Q&A]
![image](https://github.com/MetaCubeX/mihomo/assets/58256481/5e822aca-489e-41c6-b173-cab035efab40)
↑这是配置文件里写的DNS模式
![image](https://github.com/MetaCubeX/mihomo/assets/58256481/76e96d2a-18b1-463e-9ac6-cbcee51155cf)
↑这是metacubexd仪表盘，host下面的dnsMode为normal....

**Comment by xiandu999-yu** [comment]
![image](https://github.com/MetaCubeX/mihomo/assets/58256481/bc5ac9a6-3864-44f4-91d6-a6ad6bea2166)
↑singbox...估计是仪表盘的问题...？

**Reply by xiandu999-yu** [reply]
> ![image](https://private-user-images.githubusercontent.com/58256481/342110369-bc5ac9a6-3864-44f4-91d6-a6ad6bea2166.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkxNzA5NzYsIm5iZiI6MTcxOTE3MDY3NiwicGF0aCI6Ii81ODI1NjQ4MS8zNDIxMTAzNjktYmM1YWM5YTYtMzg2NC00NGY0LTkxZDYtYTZhZDZiZWEyMTY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIzVDE5MjQzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg3ZTg1NTVmNWIwN2NlYmY1ZDFiY2YzNmY3NDAzYmUyZDNiYjM2ZjZkNWM0YzVmMDZlMWE5MTk2MGI1N2UxYzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.val5dapa1SISChdKKMiNXr_XBS764PoDQvcy1t5wP7g) ↑singbox...估计是仪表盘的问题...？

不管怎样还是希望有人解答一下normal...

**Reply by qq6510** [reply]
dns泄露吧，没有走你设定的dns，应该是走原本配置的dns

**Reply by Plutonium141** [reply]
https://github.com/vernesong/OpenClash/issues/1819
